updatehub/updatehub.go

```diff
@@ -643,16 +644,26 @@ func (uh *UpdateHub) validateProcedure() error {
 
 		newActive := (active - 1) * -1
 
+		// Switch the active partion
 		setActiveErr := aii.SetActive(newActive)
 		if setActiveErr != nil {
 			return setActiveErr
 		}
 
+		// Force reboot
 		uh.Rebooter.Reboot()
 
 		return err
 	}
 
+	// We can Validate the update by setting
+	// 'upgrade_available' to 0, and then go
+	// back to the state machine.
+	setValidateErr := aii.SetValidate()
```

Comment:
I think it is more idiomatic to reuse `err` here. @gustavosbarreto ?

Reply:
```err = aii.SetValidate()```

Reply:
Sure

Reply:
I think it's consistent with the other errors handling (setActiveErr and activeErr)